.github/ISSUE_TEMPLATE/workflows/azure-pipelines_release.yml

```diff
@@ -0,0 +1,259 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+name: $(TeamProject)_$(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+# Trigger is on release branches e.g. release/v1.0.10
+trigger:
+  branches:
+    include:
+      - release/*
```

Comment:
what about triggering on tags instead? this is what we've been using in kiota/java sdk/go sdk and others, works pretty well and prepares the GitHub release

Reply:
@baywet using tags works perfectly as long we use tags for official releases (which seem to be practice in this repo https://github.com/microsoft/OpenAPI.NET.OData/tags). I will update the pipeline to trigger on tags. Thanks you.